discussment-core/src/main/java/org/danekja/discussment/core/service/imp/DefaultCategoryService.java

```diff
@@ -62,6 +62,15 @@ public void removeCategory(Category entity) throws AccessDeniedException {
         }
     }
 
+    public Category getDefaultCategory(){
+        Category category = categoryDao.getById(Category.DEFAULT_CATEGORY_ID);
+        if(category == null) {
+            category = new Category("default category");
```

Comment:
Doesnt set Category ID to Category.DEFAULT_CATEGORY_ID. May work only by coincidence.